romanesco/__init__.py

```diff
@@ -240,14 +240,14 @@ def extractId(spec):
 
             # Convert data
             if auto_convert:
-                converted = romanesco.convert(task_input["type"], d,
-                                              {"format": task_input["format"]}, **kwargs)
+                converted = romanesco.convert(
+                    task_input["type"], d,
+                    {"format": task_input["format"]},
+                    fetch=False, **kwargs)
+
```

Comment:
This is where the extra fetch was happening so I added a `fetch` parameter for `convert`.